pylint/plugins/hass_enforce_class_module.py

```diff
@@ -65,7 +65,8 @@
         "WeatherEntityDescription",
     },
 }
-_PLATFORMS: set[str] = {platform.value for platform in Platform}
+_ENTITY_COMPONENTS: set[str] = {platform.value for platform in Platform}
+_ENTITY_COMPONENTS.add("tag")
```

Comment:
Hmmm why is Tag added here?

Shouldn't that be in the `Platform` enum?

Reply:
Tag doesn't support platforms.

Reply:
Side note: There are more entity components that don't support platforms.

Reply:
Quick scan of PLATFORM_SCHEMA_BASE shows that maybe `conversation` , `notify` and `stt` should be removed from Platform enum?

Reply:
No, they all support platforms. They don't support YAML and only config entries.

Reply:
Thanks! I'm getting regularly confused between platform/component/integration/etc.